<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>254</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Source Sans Pro</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Speechless</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="mainTitleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Unispace</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Speechless&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <property name="spacing">
        <number>15</number>
       </property>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="pushToTalkRadioButton">
         <property name="font">
          <font>
           <family>Unispace</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Push-to-talk mode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="globalMuteRadioButton">
         <property name="font">
          <font>
           <family>Unispace</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Global Mic Mute</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionKeybinding"/>
    <addaction name="actionAutorun"/>
    <addaction name="actionStart_muted"/>
    <addaction name="actionMinimize_to_tray"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Devices</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNotifications">
    <property name="title">
     <string>Notifications</string>
    </property>
    <addaction name="actionShow_Notifications"/>
    <addaction name="actionPlay_Sounds"/>
    <addaction name="actionCustom_Sounds"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuDevices"/>
   <addaction name="menuNotifications"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAutorun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autorun at startup</string>
   </property>
  </action>
  <action name="actionKeybinding">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Keybinding</string>
   </property>
  </action>
  <action name="actionShow_Notifications">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Notifications</string>
   </property>
  </action>
  <action name="actionPlay_Sounds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play Sounds</string>
   </property>
  </action>
  <action name="actionCustom_Sounds">
   <property name="text">
    <string>Custom Sounds</string>
   </property>
  </action>
  <action name="actionStart_muted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start muted</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionMinimize_to_tray">
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
